providers/africastalking/src/lib/africastalking.provider.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  ChannelTypeEnum,
+  ISendMessageSuccessResponse,
+  ISmsOptions,
+  ISmsProvider,
+} from '@novu/stateless';
+
+import AfricasTalking from 'africasTalking';
+
+export class AfricastalkingSmsProvider implements ISmsProvider {
+  id = 'africasTalking';
+  channelType = ChannelTypeEnum.SMS as ChannelTypeEnum.SMS;
+  private africasTalkingClient: AfricasTalking.SMS;
```

Comment:
We should keep consistency in the naming.
If the brand is `Africastalking` then when using it for variable names we should keep it. If the brand is `AfricasTalking` keep that one. So far you have a mix of both implementations. Unifying it helps autocompletions when developing and improves readability.